vue/src/TestResultsPage/TestResultsPage.vue

```diff
@@ -0,0 +1,78 @@
+<!--
+  Matomo - free/libre analytics platform
+
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <div>
+    <div v-content-intro>
+      <h2>{{ translate('SecurityInfo_SecurityInformation') }}</h2>
+      <p>{{ translate('SecurityInfo_PluginDescription') }}</p>
+      <p>
+        Learn more: read our guide
+        <a rel='noreferrer' target='_blank' href='https://matomo.org/docs/security/'>
+          Hardening Matomo: How to make Matomo and your web server more secure?
+        </a>
+      </p>
+
+      <p class="alert-info alert">
+        Did you know? The
+        <a target='_blank' href='https://plugins.matomo.org/ActivityLog' rel="noreferrer noopener">
+          Activity Log plugin for Matomo
+        </a> improves your security and diagnostic by letting you quickly review the actions
```

Comment:
This is currently displayed that way: 
![image](https://user-images.githubusercontent.com/1579355/187914140-abb8ebd7-338c-40d4-b16b-5551273bf538.png)

Guess the spacing / line breaks here needs to be adjusted so the space isn't underlined.

Reply:
@sgiehl should be fixed